modin/engines/ray/utils.py

```diff
@@ -111,6 +110,17 @@ def initialize_ray(
                 logging_level=100,
             )
         else:
+            from modin.error_message import ErrorMessage
+
+            # This string is intentionally formatted this way. We want it indented in
+            # the warning message.
+            ErrorMessage.not_initialized(
+                "Ray",
+                """
```

Comment:
Very minor, but would this look better if the quotes weren't indented?

Reply:
Black did this. The first quotes are indented but after that the string is interpreted literally, so the last quotes don't have an indent otherwise the indent would be reflected in the message.

Reply:
Ah ok i guess there is no choice in the matter then.